src/Skoruba.IdentityServer4.Admin/Program.cs

```diff
@@ -7,11 +8,11 @@ namespace Skoruba.IdentityServer4.Admin
 {
     public class Program
     {
-        public static void Main(string[] args)
+        public static async Task Main(string[] args)
         {
             var host = BuildWebHost(args);
 
-            //DbMigrationHelpers.EnsureSeedData(host);
+            //await DbMigrationHelpers.EnsureSeedData(host);
```

Comment:
Instead of commenting it out you could add a build configuration which sets a variable and then `#if` it. This way it is not clear if commenting it out is intended.

Alternatively add a comment above this line saying when to comment out.